stdlib/private/StdlibCollectionUnittest/CheckCollectionType.swift.gyb

```diff
@@ -1549,24 +1573,53 @@ extension TestSuite {
     func toCollection(_ r: CountableRange<Int>) -> C {
       return makeCollection(r.map { wrapValue(OpaqueValue($0)) })
     }
+
+%   for i in ['index', 'formIndex']:
```

Comment:
In general, I would prefer a more descriptive name. But I guess it's alright for a scope of 20 lines.